package.json

```
@@ -45,7 +44,7 @@
     "json-loader": "0.5.7",
     "lodash.debounce": "4.0.8",
     "lodash.defaults": "4.2.0",
-    "node-sass": "4.14.1",
+    "node-sass": "^7.0.1",
```

Generally on Scratch we use a specific version for dependencies, and then use something like renovate to submit dependency update PRs to avoid surprises when different people build the site at different times.

---

I tried to bump all dependencies to latest and that opened up a whole different world of issues. 😭

---

I agree this is a stop gap, but we should still remove the `^`, once it gets into the file, there's a chance it stays.
```suggestion
    "node-sass": "7.0.1",
```

---

Yep. I'll pin it. All good. I was chatting with CWF. I'll look into renovate. I don' think this PR doesn't need to move past develop for now unless you think otherwise.